smartcontractkit/external-adapters-js
# Pull adapter changeset check into separate Github action to avoid running for PRs against master branch

## Description
Currently, the Github Action to check whether a PR containing adapter changes runs on release PRs against the `master` branch. Release PRs against `master` do not have changeset files as they have been consumed in the previous "Consume changesets" step of the release process. This change alters the action to only run on PRs against the `develop` branch.

## Changes
- Move adapter changeset check out of "main" Github Action

## Steps to Test
1. Create a PR against `develop` branch
2. Observe that the "Adapter changes accompanied by a changeset" Github Action runs
3. Create a PR against `develop` branch
4. Observe that the "Adapter changes accompanied by a changeset" Github Action does not run

Example PR against `master` branch: https://github.com/smartcontractkit/external-adapters-js/pull/1546

## Quality Assurance
